### ISSUE TITLE
Customize the HTML template

### ISSUE THREAD

**Issue by tmbb** [PragTob/benchee_html#40]
I'm thinking of including some benchmarks as part of the docs as a way of justifying some implementation choices and a s a teaching tool on how to extend my library.

You can access Benchee's HTML pages form the README ([here](https://tmbb.github.io/makeup_elixir/readme.html#benchmarks)). I'd like to change the template so that I could add other information, such as a title for the benchmark.

**Comment by PragTob** [comment]
Hi there,

adding the title as you said in PragTob/benchee#208 seems a good idea. Other information, I'm not sure about. What kind of information would that be?

If it's useful for everyone, let's add it for everyone. Adding custom pieces of HTML seems hard(er) as you also need to specify where it's rendered and when it's rendered and that seems like a lot of overhead. If you have really custom requirements, implementing your own benchee formatter might probably be the better option.

**Comment by tmbb** [comment]
I agree. I don't need to add the extra link I was thinking about. I just
need to use the browser back button.
